index.html

```diff
@@ -1,203 +1,349 @@
+                            <div class="item-title">
+                                <h1>Mult-Post Stories</h1>
+                                <h2>Gain + Glory</h2>
+                            </div>
+                            <ul class="technologies">
+                                <li>Javascript</li>
+                                <li>Ruby on rails</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                            </ul>
+                        </div>
+                        <div class="green-btn">
+                            <a href="single-project.html">See Project</a>
+                        </div>
+                    </div>
+
+                    <div class="grid-item">
+                        <div class="item-image">
+                            <!-- <img src="" alt="place image here" /> -->
+                        </div>
+                        <div class="item-description">
+                            <div class="item-title">
+                                <h1>Mult-Post Stories</h1>
+                                <h2>Gain + Glory</h2>
+                            </div>
+                            <ul class="technologies">
+                                <li>Javascript</li>
+                                <li>Ruby on rails</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                            </ul>
+                        </div>
+                        <div class="green-btn">
+                            <a href="single-project.html">See Project</a>
+                        </div>
                     </div>
+
                 </div>
 
-                <div class="project-grid-item">
-                    <div class="item-image">
-                        <!-- <img src="" alt="place image here" /> -->
-                    </div>
-                   <div class="item-description">
-                    <div class="item-title">
-                        <h1>Mult-Post Stories</h1>
-                        <h2>Gain + Glory</h2>
-                    </div>
-                    <ul class="technologies">
-                        <li>Javascript</li>
-                        <li>Ruby on rails</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                    </ul>
-                   </div>
+            </section>
+            <section class="section">
+                <div class="about-bg-imgs">
+                    <img src="assets/images/about-me/about-me-left.svg" alt="bg-image"/>
+                    <img src="assets/images/about-me/aboutmedesktop-right.svg" alt="bg-image" />
+               </div>
+                <div class="about-header">
+                    <div class="title">
+                        <h1 class="primary-text">About Me</h1>
+                        <hr class="line" />
+                    </div>
```

Comment:
- [ ] The `<h1>` tag is inappropriate in this case because we have only 1 `<h1>` tag on a page and that should be for the main heading. Every other header on the page should be between `<h2>` and `<h6>`. Please apply this change 🙏